src/odemis/acq/test/fastem_test.py

```diff
@@ -39,9 +39,9 @@
 from odemis.util import test, img
 
 # * TEST_NOHW = 1: not connected to anything => skip most of the tests
-# * TEST_NOHW = 0: connected to the real hardware or simulator
+# * TEST_NOHW = 0: connected to the real hardware or simulator (backend needs to be running)
 # technolution_asm_simulator/simulator2/run_the_simulator.py
-TEST_NOHW = os.environ.get("TEST_NOHW", "0")  # Default is HW/simulator testing
+TEST_NOHW = (os.environ.get("TEST_NOHW", 0) != 0)  # Default is HW
```

Comment:
I think both works as in both cases you compare either two strings with each other and check they are the same or two int. The result will be in both cases True or False.

Reply:
I think it will not work if your environment variable TEST_NOHW is set to "0". Then you'll get `os.environ.get("TEST_NOHW", 0) = "0"` and `"0" != 0 =True` while you would want it to be False.

Reply:
Good point. What do we usually do? Set it to "0" or 0?  I am then also confused why we have both versions in Odemis. How is it handled in the script we use to run the tests?

Reply:
Indeed, "0" is the right way.